src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_help.py

```diff
@@ -152,7 +152,7 @@
 short-summary: Start protecting a previously unprotected Azure VM as per the specified policy to a Recovery services vault.
 examples:
   - name: Start protecting a previously unprotected Azure VM as per the specified policy to a Recovery services vault. (autogenerated)
-    text: az backup protection enable-for-vm --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault --vm $(az vm show
+    text: az backup protection enable-for-vm --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault --vm $(az vm show)
```

Comment:
Something wrong with the autogen here. `az vm show` will simply not work. It requires a name and resource group. Furthermore, to be used in this fashion, it would also need `--query id -o tsv`